frontend/package-lock.json

```diff
@@ -104,12 +104,45 @@
         "@glimmer/di": "^0.2.0"
       }
     },
+    "@types/acorn": {
```

Comment:
not sure why u need this file in commit?

Reply:
Removed in Fixed in 92c4f72